contracts/script/OPSuccinctDeployer.s.sol

```diff
@@ -17,7 +17,7 @@ contract OPSuccinctDeployer is Script, Utils {
         OPSuccinctL2OutputOracle oracleImpl = new OPSuccinctL2OutputOracle();
         Proxy proxy = new Proxy(msg.sender);
 
-        upgradeAndInitialize(address(oracleImpl), config, address(proxy), true);
+        upgradeAndInitialize(address(oracleImpl), config, address(proxy), true, address(0));
```

Comment:
There's also a case where users may not have an L2OO Proxy but want to re-use the implementation logic that already exists.

E.g. They want to deploy an OP Succinct chain on Mainnet, and there's an implementation contract that's already been deployed.

This forge script should be logic for deploying a new proxy, and should support the `OP_SUCCINCT_L2_OUTPUT_ORACLE_IMPL` varaible.

Can you also configure this script to support `adminPk` and `deployPk`? Currently, it just sets the owner of the Proxy to `msg.sender`, when it should use the account associated with `admin_pk` if it's set.